src/MCPClient/lib/clientScripts/archivematicaClamscan.py

```diff
@@ -23,7 +23,6 @@
 
 from __future__ import print_function
 
-import os
```

Comment:
Someone enabled CI, and our `.travis.yml` came into effect and tox started complaining about linting issues. I did address them in a separate commit though to avoid confusion.